CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## 0.11.2dev
+- [Fix] Adds input validation with suggestions to `plot.grid_search` (#53)
```

Comment:
let's pin ploomber-core>=0.2.6 in setup.py since that's where we introduced the change https://github.com/ploomber/core/blob/main/CHANGELOG.md#026-2023-02-16